Getting-Started/Code/Debugging/Logging/index-v8.md

```diff
@@ -0,0 +1,250 @@
+<add key="serilog:minimum-level:override:Microsoft.AspNetCore.Mvc" value="Error" />
+<add key="serilog:minimum-level:override:YourNameSpace" value="Information" />
+```
+
+:::warning
+If you change the **serilog:minimum-level** to be **Error** then the following example above would only log messages from *Microsoft.AspNetCore.Mvc* and not any warning, info or debug messages from the *Microsoft* namespace
+:::
+
+### Writing your own log messages to a custom file
+Add the following to the `/config/serilog.user.config` file, which will create a new JSON log file
+
+```xml
+<!-- Write to a user log file -->
+<add key="serilog:using:File" value="Serilog.Sinks.File" />
+<add key="serilog:write-to:File.path" value="%BASEDIR%\my-logs\my-custom-logfile.txt" />
+<add key="serilog:write-to:File.shared" value="true" />
+<add key="serilog:write-to:File.restrictedToMinimumLevel" value="Debug" />
+<add key="serilog:write-to:File.retainedFileCountLimit" value="32" /> <!-- Number of log files to keep (or remove value to keep all files) -->
+<add key="serilog:write-to:File.rollingInterval" value="Day" /> <!-- Create a new log file every Minute/Hour/Day/Month/Year/infinite -->
+```
+
+### Filtering user log file to include only log messages from your namespace
+With the above example we are able to write to a seperate JSON log file, but adding these additional lines to `serilog.user.config` will allow you to filter and include log messages. For further details on specific expressions you can write, refer to the readme of the [Serilog Filters Expression project](https://github.com/serilog/serilog-filters-expressions)
+
+```xml
+<!-- Filters all sink's in the serilog.user.config to use this expression -->
+<!-- Common use case is to include SourceType starting with your own namespace -->
+<add key="serilog:using:FilterExpressions" value="Serilog.Filters.Expressions" />
+<add key="serilog:filter:ByIncluding.expression" value="StartsWith(SourceContext, 'MyNamespace')" />
+```
+
+### Writing log events to different storage types
+Serilog has a similar concept to Log4Net with its appenders, which are referred to Serilog Sinks.
+A Serilog Sink, allows you to persist the structured log message to a data store of your choice. In v8.0+ we use the *Serilog.Sinks.File* to allow us to write a .txt or .json file to disk. But the Serilog project anmd the wider Serilog community allows you to store these logs in various locations.
+
+Here are some examples:
+* [Azure Table Storage](https://github.com/serilog/serilog-sinks-azuretablestorage)
+* [Elastic Search](https://github.com/serilog/serilog-sinks-elasticsearch)
+* [Elmah.io](https://github.com/serilog/serilog-sinks-elmahio)
+* [Seq](https://github.com/serilog/serilog-sinks-seq)
+* [SQL Server](https://github.com/serilog/serilog-sinks-mssqlserver)
+* [Windows Event Log](https://github.com/serilog/serilog-sinks-eventlog)
+* [Many more](https://github.com/serilog/serilog/wiki/Provided-Sinks)
```

Comment:
I would add the least possible to this list.  They can look at all the provided sinks on the serilog page.  Otherwise this will become a magnet for link builders or "fans" adding their product.  I (e.g.) would add Exceptionless instead of Elmah.

Seq might be the only exception (as we are running Seq too...)

Reply:
Good point, updated this @dampee